airflow/utils/db.py

```diff
@@ -950,8 +950,12 @@ def synchronize_log_template(*, session: Session = NEW_SESSION) -> None:
         log.info("Log template table does not exist (added in 2.3.0); skipping log template sync.")
         return
 
+    es_log_it_template_fallback = "{dag_id}-{task_id}-{execution_date}-{try_number}"
+
     filename = conf.get("logging", "log_filename_template")
-    elasticsearch_id = conf.get("elasticsearch", "log_id_template")
+    elasticsearch_id = conf.get("elasticsearch", "log_id_template", fallback=es_log_it_template_fallback)
+
+    # TODO: The elasticsearch specific stuff here is probably inappropriate - provider is bleeding into core
```

Comment:
Oh. Absolutely -we have still a few remnants of it and I hope when I go with #44511 we can get rid of all those remnants.